vispy/app/backends/_wx.py

```diff
@@ -195,7 +195,7 @@ def Skip(self):
         pass
 
 
-class CanvasBackend(Frame, BaseCanvasBackend):
```

Comment:
Whatever we did to protect the `Frame` subclassing should work for your `GLCanvas` case.